central/debug/service/service_test.go

```diff
@@ -168,3 +177,64 @@ func (s *DebugServiceTestSuite) TestGetConfig() {
 	s.NoError(err)
 	s.Equal(expectedConfig, actualConfig)
 }
+
+func (s *DebugServiceTestSuite) TestGetBundle() {
+	stubTime := time.Date(2023, 03, 14, 0, 0, 0, 0, time.UTC)
+	now = func() time.Time {
+		return stubTime
+	}
+
+	w := mockResponseWriter{
+		code: http.StatusOK,
+		data: bytes.NewBuffer(nil),
+	}
+	testutils.SetVersion(s.T(), testutils.GetExampleVersion(s.T()))
+	db := mocks.NewMockDB(s.mockCtrl)
+	pgxRows := pgxpoolmock.NewRows([]string{"server_version"}).AddRow("15.1").ToPgxRows()
+	pgxRows.Next() // This is a workaround for https://github.com/pashagolub/pgxmock/issues/133
+	db.EXPECT().QueryRow(gomock.Any(), "SHOW server_version;").Return(pgxRows)
+	globaldb.GetPostgresTest(s.T(), db)
```

Comment:
I think the name of this function can be misleading, as it actually sets postgres instance